[16] Technical CoreData/Technical CoreData/Views/ContentView.swift

```diff
@@ -12,13 +12,48 @@ struct ContentView: View {
 	@Environment(\.managedObjectContext)
 	var managedObjectContext
 	
-	
-	
     var body: some View {
-//		WizardsList()
-//		StarshipsList()
-//		SingersList()
-		CandyBarsList()
+		NavigationView {
```

Comment:
Love this navigation view. Much better than the original tutorial code showing a screen each time